homeassistant/components/cover/myq.py

```diff
@@ -97,6 +98,11 @@ def is_closed(self):
         """Return true if cover is closed, else False."""
         return MYQ_TO_HASS[self._status] == STATE_CLOSED
 
+    @property
+    def is_open(self):
```

Comment:
This property doesn't exist in the base class. Just remove this. It's enough to extend the dict above.

Reply:
Not sure I fully grasp, are you saying removing this:

    @property
    def is_open(self):
        """Return true if cover is open, else False."""
        return MYQ_TO_HASS[self._status] == STATE_OPEN

would have the same effect?